lib/elixir_google_scraper/scraper/keyword_scraper.ex

```diff
@@ -0,0 +1,29 @@
+defmodule ElixirGoogleScraper.Scraper.KeywordScraper do
+  alias ElixirGoogleScraper.Scraper.SearchResponseParser
+
+  @user_agent "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.447"
+
+  def scrape(keyword) do
+    response = make_request(keyword)
+    SearchResponseParser.parse(response.body)
+  end
+
+  defp google_search_url, do: "https://www.google.com/search"
```

Comment:
I think maybe we can move this after to `build_url` so we don't need to jump around too much. Or another approach is making this a constant on the top.